### PR TITLE
Bump yarn 1.6.0

### PR DESCRIPTION
**Pull request by cdwills**
- we like yarn, we like updated and consistent yarn best.
- bump yarn to 1.6.0, and lock it there.
- add this to our new alpine images.

### CODE REVIEW

File: 6-alpine/Dockerfile [hunk 1]
```diff
@@ -2,9 +2,28 @@ FROM node:6-alpine
...
+  done \
+  && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
+  && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
+  && gpg --batch --verify yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
+  && mkdir -p /opt \
+  && tar -xzf yarn-v$YARN_VERSION.tar.gz -C /opt/ \
+  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarn /usr/local/bin/yarn \
+  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarnpkg /usr/local/bin/yarnpkg \
+  && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
+  && apk del .build-deps-yarn
```
Comment (dbertram) [inline 6-alpine/Dockerfile]: This newly added snippet differs from the one we use in `4/Dockerfile`, `6/Dockerfile`, etc. Is that just due to `alpine` vs `debian`?

**Comment by kdstew** [comment]
@cdwills do we still need `npm install --global yarn` here https://github.com/articulate/docker-articulate-node/pull/31/files#diff-b0c0d39856937befc8cfc9ee67bf4abaR29

File: 6-alpine/Dockerfile (hunk 1 above)
Comment (cdwills) [reply 6-alpine/Dockerfile]: @dbertram yes, it is `alpine` vs `debian`.  I got that snippet from the official docker node alpine builds.

**Comment by cdwills** [comment]
@kdstew you are correct, good catch, I'll remove.

**Comment by cdwills** [comment]
image

**Pull request merged**